cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/CruiseControl/CruiseControlApi.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.cluster.operator.assembly.CruiseControl;
+
+import io.vertx.core.Future;
+
+/**
+ * Cruise Control REST API interface definition
+ */
+public interface CruiseControlApi {
+
+    Future<CruiseControlResponse> getCruiseControlState(boolean verbose);
+    Future<CruiseControlResponse> rebalance(RebalanceOptions options);
+    Future<CruiseControlResponse> getUserTaskStatus(String userTaskId);
+    Future<CruiseControlResponse> stopExecution();
```

Comment:
does it stop just the ongoing operation right? because CC can run one at time. Or we need to pass the uuid (maybe in the header)? I don't see parameters for this in the REST wiki page.

Reply:
This end point takes no parameters (other than JSON) and stops the (only) running task.